tests/Localization/LvLvTest.php

```diff
@@ -21,119 +21,119 @@ class LvLvTest extends LocalizationTestCase
     public const CASES = [
         // Carbon::parse('2018-01-04 00:00:00')->addDays(1)->calendar(Carbon::parse('2018-01-04 00:00:00'))
         // 'Tomorrow at 12:00 AM',
-        'Tomorrow at 00:00',
+        'rīt plkst. 00:00',
```

Comment:
lv_LV will never be different from just lv, it is the same exact dialect. Why the duplication?

Reply:
Multiple reasons:
- Consistency with all other locales (either we refactor them all or none, but I want to avoid a mix)
- When future changes will apply, we'll be able to see the before/after difference for each impacted locale.
- "will never", let's say Latvia is an immutable country with immutable borders and immutable languages, but the generic idea is to consider languages and regions evolve. And while there are countries having only 1 official language and this language being official in only 1 country, we also have plenty of languages being official in multiple countries and countries having multiple official languages. For instance, while `de_DE` represent the official German language as spoken in Germany, `de` represent the German more common base across all regions speaking German. If Germany officially change its date format, we'll update `de_DE` but not `de`.
- All localization tests follow the same pattern, so you can easily compare outputs of 2 locales, and regenerate them by scripting.
- Last generic concept is to avoid to mirroring source code in tests code, as it increases the risk of mirroring errors. For instance: `assert(4, sum(2, 2))` over `assert(2 + 2, sum(2, 2))`